pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -568,3 +559,13 @@ func (c *AcceptNewlyObservedReadyPods) Accept(deployment *kapi.ReplicationContro
 	}
 	return nil
 }
+
+// deployerLog logs the deployer messages to standard output for standard verbosity level
+// and use glog facility to log messages when the log level is verbose.
+func deployerLog(out io.Writer, message string, v ...interface{}) {
+	if glog.V(4) {
```

Comment:
@kargakis when using higher log level, the output is messed up by combining the "fancy" messages with generic glog messages. This will make it look much more consistent.
It also allow you to add timestamps in deployer log messages if you want.